### PR TITLE
Added --skip-validation option

### LINKED ISSUE #32: Skipping JS files validation
We need that for `preprocess-plugin` for obfuscated plugins like `spreadsheet`.

### PR DESCRIPTION
**Pull request by vokiel**
Closes #32

### CODE REVIEW

**Review by f1ames — CHANGES_REQUESTED** [review]
If I add spreadsheet plugin to build config like:

```
spreadsheet: 1
```

and run build with:

```sh
./build.sh standard --skip-validation 2
```

The validation error is still there :thinking: 

image

Same with (without spreadsheet in config):

```sh
./build.sh standard all --skip-validation 2
```

And with:

```sh
./build.sh standard --skip-validation 2 -r spreadsheet
```

Tried also with `--skip-validation=2` but the result is the same :thinking: And it seems the `jar` is fine since the options is recognized correctly (no error about unrecognized option).

@vokiel any ideas? :thinking:

File: src/lib/controller.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ CKBuilder.Controller = function() {
 		[ null, "leave-js-unminified", false, "leave javascript files as is, do not minify them" ],
 		[ null, "leave-css-unminified", false, "leave CSS files as is, do not minify them" ],
 		[ "s", "skip-omitted-in-build-config", false, "exclude from release all plugins/skins that are not specified in build-config" ],
+		[ null, "skip-validation", [ "LEVEL", "skip validation level (0, 1, 2)." ], "sets the skip validation level" ],
```
[the remark this answers is not in the document]
Comment (f1ames) [reply src/lib/controller.js:39]: Ok, now I see level 1 checks for non-existing plugin files only but doesn't validate JS.

[the remark this answers is not in the document]
Comment (f1ames) [reply src/lib/controller.js:39]: While `0` does both checks and `2` skips both checks.

**Comment by f1ames** [comment]
I had an aha moment suddenly and realized that I was passing `--skip-validation` flag to `build.sh` script instead of `ckbuilder.jar` directly... so called "Friday evening effect"... :sweat\_smile::facepalm::see\_no\_evil: 

So ignore my previous [https://github.com/ckeditor/ckbuilder/pull/33#pullrequestreview-569412882](https://github.com/ckeditor/ckbuilder/pull/33#pullrequestreview-569412882) comment as I need to take another look on this PR :crossed\_fingers:

**Comment by vokiel** [comment]
We use it like this:

```shell
java -jar ckbuilder.jar --no-ie-checks --preprocess-plugin --leave-js-unminified --skip-validation=1 --overwrite /path_to_plugin
```

**Review by f1ames — COMMENTED** [review]
Thanks @vokiel for the [https://github.com/ckeditor/ckbuilder/pull/33#issuecomment-762070370](https://github.com/ckeditor/ckbuilder/pull/33#issuecomment-762070370) - I was able to figure out how to use it :+1: 

Using it as you mentioned works as expected. So I was able to use:

```shell
java -jar ckbuilder.jar --no-ie-checks --preprocess-plugin --leave-js-unminified --skip-validation=0\1\2 --overwrite /path_to_plugin
```

and it worked as expected.

What made me confused was the fact that it works only with `--preprocess-plugin` (that's my assumption) because running it with the way presets use `ckbuilder.jar`:

```
java -jar ckbuilder/2.4.2/ckbuilder.jar --build ckeditor build/4.15.2/standard -s  --version="4.15.2 (Standard)" --revision ece98ca43 --build-config presets/standard-build-config.js --skip-validation=0/1/2 --no-zip --no-tar --overwrite  standard
```

always failed due to validation. So it means **this flag can be only used when pre-processing single plugin and not building entire editor. Is it correct @vokiel?** :thinking: 

This is basically how we use it and what we need so I'm fine with it. It will be good to mention in the option description (I can take care of that).

**Review by f1ames — APPROVED** [review]
LGTM :+1: :clap: 

We have adjusted option name and its description to be more meaningful.

**Pull request merged**